src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -105,13 +136,13 @@ public void start(Stage primaryStage) throws Exception {
         // menuBar.getMenus().addAll(menuConvo);
 
         ObservableList<String> usersList = FXCollections.observableArrayList(
-                 "Julia", "Ian", "Sue", "Matthew", "Hannah", "Stephan", "Denise");
+                "Julia", "Ian", "Sue", "Matthew", "Hannah", "Stephan", "Denise");
```

Comment:
Definitely make use of the interface that we established to make the name placement more dynamic.